packages/react-tinacms-github/src/toolbar-plugins/useGithubToolbarPlugins.tsx

```diff
@@ -16,22 +16,27 @@ limitations under the License.
 
 */
 
-import { Form, useCMS } from 'tinacms'
+import { useCMS } from 'tinacms'
 import { useEffect } from 'react'
 import { getForkName } from '../github-editing-context/repository'
 import { PRPlugin } from './pull-request'
 import { ForkNamePlugin } from './ForkNamePlugin'
+import { GithubClient } from '../github-client'
 import { Plugin } from 'tinacms'
+import { useGithubEditing } from 'github-editing-context'
 
-export const useGithubToolbarPlugins = (
-  form: Form<any>,
-  editMode: boolean,
-  baseRepo: string,
-  baseBranch: string = 'master'
-) => {
+export const useGithubToolbarPlugins = () => {
   const cms = useCMS()
+  const { editMode } = useGithubEditing()
+  const github: GithubClient = cms.api.github
+  const baseRepo = github?.baseRepoFullName
+  const baseBranch = github?.baseBranch
 
   useEffect(() => {
+    if (!baseRepo || !baseBranch) {
+      // Warn dev appropriately
```

Comment:
Shall we throw an error here? (We'll also have to check if it's in editMode)

Reply:
This was removed in the other PR